src/styles/index.scss

```diff
@@ -1,3 +1,15 @@
+@import './abstracts/variables';
```

Comment:
hm interesting why you choose such naming _abstracts_
As for me utils suits better ;)

Reply:
I just started doing the task in the middle of learning Sass before I saw that lector use /utils folder. So I googled the best practices and saw that using /abstracts folder is also a common practice and left it as it is :grin: